src/guide/receive_onchain.md

```diff
@@ -102,7 +102,7 @@ if swapInfo, err := sdkServices.ReceiveOnchain(); err != nil {
 ```cs
 try 
 {
-    var swapInfo = sdk.ReceiveOnchain();
+    var swapInfo = sdk.ReceiveOnchain(new ReceiveOnchainRequest(null));
```

Comment:
Do we have to pass null here? We use default value in the udl.

Reply:
Yes, we do:
<img width="817" alt="Scherm­afbeelding 2023-09-03 om 20 45 47" src="https://github.com/breez/breez-sdk-docs/assets/15966593/3c676bb4-3e7b-4e58-9151-cba0dd5c8ed7">

Reply:
It is in this PR that is going to be merged with the new docs:
https://github.com/breez/breez-sdk/pull/416/files#diff-5f8d23b285fe7996da0509e9fbc2496bfc9d3ea4040a4702cbe85cfc38be09eaR314

Reply:
Ah, great

Reply:
Done